baker/GrapherBaker.tsx

```diff
@@ -432,7 +431,7 @@ export const bakeAllChangedGrapherPagesVariablesPngSvgAndDeleteRemovedGraphers =
         )
 
         const progressBar = new ProgressBar(
-            "bake grapher page [:bar] :current/:total :elapseds :rate/s :etas :name\n",
```

Comment:
It's not clear this is an ETA in the output, which is already cluttered, and doesn't seem useful.